src/main/java/com/padmaja/stringreverse/stringUtil.java

```diff
@@ -0,0 +1,16 @@
+package com.padmaja.stringreverse;
+import java.lang.String;
+
+public class stringUtil {
+    public String reverse(String inputString) {
+        char[] myarray= inputString.toCharArray();
+        for(int i=0,j = inputString.length()-1; i<j; i++,j--){
+            char temp = myarray[i];
+            myarray[i]= myarray[j];
+            myarray[j]= temp;
+        }
+        return new String(myarray);
```

Comment:
why can not you return myarray.toString()

Reply:
Shallow copy is a bit-wise copy of an object. A new object is created that has an exact copy of the values in the original object. If any of the fields of the object are references to other objects, just the reference addresses are copied i.e., only the memory address is copied.(source : here)